mysql-test/suite/innodb/t/percona_changed_page_bmp_1364315.test

```diff
@@ -6,9 +6,7 @@
 
 --let $TMPDATADIR= $MYSQLTEST_VARDIR/tmpdatadir_1364315
 
---exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
---shutdown_server 10
---source include/wait_until_disconnected.inc
+--source include/shutdown_mysqld.inc
 --mkdir $TMPDATADIR
 --enable_reconnect
 --echo 1st restart
```

Comment:
It would result in $TMPDATADIR absolute path recorded in the test output. 5.7 can do --replace_result $FOO FOO for start_mysqld.inc, but this one cannot